src/physics/circular_filament.rs

```diff
@@ -1,19 +1,88 @@
 //! Magnetics calculations for circular current filaments.
+use std::num::NonZeroUsize;
+
+use rayon::{
+    iter::{IndexedParallelIterator, ParallelIterator},
+    slice::{ParallelSlice, ParallelSliceMut},
+};
+
 use crate::math::{ellipe, ellipk};
 
 use crate::{MU0_OVER_4PI, MU_0};
 
+/// Flux contributions from some circular filaments to some observation points, which happens to be
+/// the Green's function for the Grad-Shafranov elliptic operator, $\Delta^{\*}$.
+/// This variant of the function is parallelized over chunks of observation points.
+///
+/// # Arguments
+///
+/// * `ifil`: (A) current in each filament, length `m`
+/// * `rfil`:    (m) r-coord of each filament, length `m`
+/// * `zfil`:    (m) z-coord of each filament, length `m`
+/// * `rprime`:  (m) r-coord of each observation point, length `n`
+/// * `zprime`:  (m) z-coord of each observation point, length `n`
+/// * `out_psi`: (Wb) or (H) or (T-m^2) or (V-s), poloidal flux at observation locations, length `n`
```

Comment:
A weber is a henry ampere, not a henry.
```suggestion
/// * `out_psi`: (Wb) or (H-A) or (T-m^2) or (V-s), poloidal flux at observation locations, length `n`
```